src/App.tsx

```diff
@@ -32,7 +32,8 @@ import OneSignalManager from './services/oneSignal/OneSignalManager'
 
 library.add(fas, fab, far)
 
-function App (): JSX.Element {
+// eslint-disable-next-line @typescript-eslint/space-before-function-paren
+function App(): JSX.Element {
```

Comment:
Why don't you simply put back the space that was there? Is it incompatible with another linter that we have?

Reply:
problem of eslint